_posts/en/newsletters/2023-11-08-newsletter.md

```diff
@@ -0,0 +1,249 @@
+      After being registered (by calling `RegisterSharedValidationInterface()`),
+      any implemented `CValidationInterface` method will be executed each time
+      the method callback is fired using `CMainSignals`.
+      Callbacks are fired whenever the corresponding event occurs."
+  a4link="https://bitcoincore.reviews/28368#l-90"
+
+  q5="[`BlockConnected`][BlockConnected] and [`NewPoWValidBlock`][NewPoWValidBlock]
+      are different callbacks.
+      Which one is asynchronous and which one is synchronous? How can you tell?"
+  a5="`BlockConnected` is asynchronous; `NewPoWValidBlock` is asynchronous.
+      Asynchronous callbacks queue an \"event\" to be run later within the
+      `CScheduler` thread."
+  a5link="https://bitcoincore.reviews/28368#l-105"
+
+  q6="In [commit 4986edb][], why are we adding a new callback,
+      `MempoolTransactionsRemovedForConnectedBlock`, instead of using
+      `BlockConnected` (which also indicates a transaction being removed
+      from the mempool)?"
+  a6="The fee estimator needs to know when transactions are removed from the
+      mempool for any reason, not just when a block is connected.
+      Also, the fee estimator needs a transaction's base fee, and this isn't
+      provided via `BlockConnected` (which provides a `CBlock`).
+      We could add the base fee to the `block.vtx` (transaction list) entries,
+      but it's undesirable to change such an important and ubiquitous data
+      structure just to support fee estimation."
+  a6link="https://bitcoincore.reviews/28368#l-130"
+
+  q7="Why don’t we use a `std::vector<CTxMempoolEntry>` as a parameter of
+      `MempoolTransactionsRemovedForBlock` callback?
+      This would eliminate the requirement for a new struct type to hold the
+      per-transaction information needed for fee estimation."
+  a7="The fee estimator doesn't need all of the fields from `CTxMempoolEntry`."
+  a7link="https://bitcoincore.reviews/28368#l-159"
+
+  q8="How is the base fee of a `CTransactionRef` computed?"
+  a8="It's the sum of the input values minus the sum of the output values.
+      However, the callback can't access the input values because they're stored
+      in the previous transaction outputs (which the callback doesn't have access to).
+      That's why the base fee is included in the `TransactionInfo` structure."
+  a8link="https://bitcoincore.reviews/28368#l-166"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
+  version of the predominant full node implementation.  Verified test
+  binaries have not yet been released as of this writing, although we
+  expect them to be published at the preceding URL shortly after
+  publication of the newsletter.  There's a brief overview to [suggested
```

Comment:
rc2 binaries are up

Reply:
Thanks @glozow just updated the text accordingly.